static/motd.interfaces.ts

```diff
@@ -24,21 +24,23 @@
 
 import {editor} from 'monaco-editor/';
 
-interface Decoration {
+type Decoration = {
     decoration: editor.IModelDecorationOptions;
     filter: string[];
     name: string;
     regex: string;
-}
+};
 
-interface Ad {
-    filter?: string;
+export type Ad = {
+    filter: string[];
```

Comment:
This was just broken enough not to break in prod

Reply:
Oh blimey!